assets/common/src/deploy.py

```diff
@@ -262,10 +263,15 @@ def main():
     # get registered model id
     if args.registration_details:
         model_info = {}
-        with open(args.registration_details) as f:
-            model_info = json.load(f)
-        model_id = model_info["id"]
-        model_name = model_info["name"]
+
+        registration_details_file = args.registration_details/"model_registration_details.json"
```

Comment:
Declare `model_registration_details.json` in constants and use it in both register and deploy model launcher scripts